### PR TITLE
fix: curator name backfill

### PR DESCRIPTION
**Pull request by Bento007**
### Reviewers
**Functional:** @seve 

**Readability:** @atolopko-czi 

---

## Changes
- add function to update curator_name in a collection based on the owner of a collection, and update all published collection to data_submission_policy_version 2.0
- connected to #1745 

## Definition of Done (from ticket)
https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1745

### CODE REVIEW

File: scripts/cxg_admin.py [hunk 1]
```diff
@@ -558,6 +558,90 @@ def add_trailing_slash_to_explorer_urls(ctx):
...
+                collection.data_submission_policy_version = "2.0"
+
+    access_token = get_auth0_access_token()
+    unique_owners = get_owners_from_database()
+    owners = dict()
+    for owner in unique_owners:
+        try:
+            owner_name = get_owner_info_from_auth0(owner, access_token)
+            if owner_name:
+                owners[owner] = owner_name
```
Comment (seve) [inline scripts/cxg_admin.py:638]: Should we do a check here that owner_name isn't an empty string?

Comment (Bento007) [reply scripts/cxg_admin.py:638]: in python an empty string == False

Comment (seve) [reply scripts/cxg_admin.py:639]: Sorry, should have elaborated, do we want to notify the user when there is an owner with no name attached to their account?

Comment (Bento007) [reply scripts/cxg_admin.py:639]: We can run this to populate all of the curators and then run another query to see what collections have no curator.

**Review by seve — APPROVED** [review]

**Pull request merged**